pkg/scanner/scan.go

```diff
@@ -139,6 +140,7 @@ func NewScanner(driver Driver, ar artifact.Artifact) Scanner {
 
 // ScanArtifact scans the artifacts and returns results
 func (s Scanner) ScanArtifact(ctx context.Context, options types.ScanOptions) (types.Report, error) {
+	startTime := time.Now().UTC()
```

Comment:
What if use [clock](https://github.com/aquasecurity/trivy/blob/main/pkg/clock/clock.go) package?
Then you can use `SetFakeTime` for `scan_test.go`.